js/src/components/paid-ads/ads-campaign.js

```diff
@@ -55,6 +61,24 @@ export default function AdsCampaign( {
 				'google-listings-and-ads'
 		  );
 
+	const { data: budgetData, loading } = useFetchBudgetRecommendationEffect(
+		formContext.values.countryCodes
+	);
+
+	const { country = '', daily_budget: dailyBudget } = getHighestBudget(
+		budgetData?.recommendations || []
```

Comment:
@ankitrox We already have a check here https://github.com/woocommerce/google-listings-and-ads/pull/2552/files#diff-3bdecec9939ad884905f627b3905c8b138f73d843890bb6d41799694c57f2ab8R10 in case `recommendations` is falsy. Shall we return an empty array in the `getHighestBudget` function instead of `null`?